mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -145,6 +164,15 @@ class LinearSVC @Since("2.2.0") (
   def setAggregationDepth(value: Int): this.type = set(aggregationDepth, value)
   setDefault(aggregationDepth -> 2)
 
+  /**
+   * Set optimizer for LinearSVC. Supported options: "lbfgs" and "owlqn".
+   *
+   * @group setParam
+   */
+  @Since("2.2.0")
+  def setOptimizer(value: String): this.type = set(optimizer, value.toLowerCase(Locale.ROOT))
```

Comment:
Maybe just call it solver, since `LinearRegression` also has something similar like this.